jobs/kubernetes-system-specs/templates/config/kubedns-rc.yml

```diff
@@ -77,58 +99,73 @@ spec:
           name: dns-tcp-local
           protocol: TCP
         volumeMounts:
-        - name: config
+        - name: kube-dns-config
           mountPath: /var/vcap/jobs/kubeconfig/config/
+          # ^^ What is this used for?
```

Comment:
The `mountPath` represents the path inside the container, to which the volume is mounted. 

Our usage of `/var/vcap` in internal paths is confusing and I suggest we change it to avoid mixing up host paths and container paths.